testing/setest/Makefile

```diff
@@ -0,0 +1,35 @@
+############################################################################
+# apps/testing/setest/Makefile
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.  The
+# ASF licenses this file to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance with the
+# License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+############################################################################
+
+# Copyright 2023 NXP
```

Comment:
remove all NXP copyright from code base

Reply:
Could you clarify this further?

Reply:
The new written code is better to declare as Apache copyright, the proprietary license is hard to accept or at least user need get the explicit notice like what we have done for BSD/MIT here:
https://github.com/apache/nuttx/blob/master/Kconfig#L28-L90

Reply:
But it's Apache 2.0 licensed, I don't see the problem here.

Reply:
but as far as I know, the new code should avoid add the company copyright like "Copyright 2023 NXP".

Reply:
It's indicating the original copyright holder, I don't see why this should be avoided. It's quite common practice in other projects to include this i.e. Linux, Zephyr. OpenCV and more
`Copyright <YEAR> <COPYRIGHT HOLDER>`

Reply:
@acassis @jerpelea please take a look.

Reply:
I don't think this is correct. If NXP is not donating the code I believe it must be treated as third party and handled via the NOTICE.